src/Compiler/GetCompiler.php

```diff
@@ -219,6 +240,67 @@ protected function readGet()
         return true;
     }
 
+    protected function readCount()
+    {
+        if (!self::scanFunction($this->request, $name, $arguments)) {
+            return false;
+        }
+
+        if ($name !== 'count') {
+            return false;
+        }
+
+        if (!isset($arguments) && (count($arguments) > 0)) {
+            throw CompilerException::badGetArgument($this->request);
+        }
+
+        // at this point they definitely intend to use a count function
+        $this->request = reset($arguments);
+
+        $columnId = $this->mysql->addCount();
+        $this->phpOutput = Output::getValue($columnId, false, Output::TYPE_INTEGER);
+
+        return true;
+    }
+
+    protected function readParameterizedAggregator($functionName)
+    {
+        if (!self::scanFunction($this->request, $name, $arguments)) {
+            return false;
+        }
+
+        if (!array_key_exists($functionName, self::$parameterizedAggregators)) {
+            return false;
+        }
+
+        if (!isset($arguments) && (count($arguments) !== 1)) {
```

Comment:
Here's a bit more information about the "isset" function:
http://php.net/manual/en/function.isset.php

It's unlikely that the condition here behaves as intended.